Lombiq.Tests.UI/EmbeddedResourceProvider.cs

```diff
@@ -0,0 +1,18 @@
+using System.IO;
+using System.Text;
+
+namespace Lombiq.Tests.UI
+{
+    internal static class EmbeddedResourceProvider
+    {
+        internal static string ReadEmbeddedFile(string fileName)
+        {
+            var assembly = typeof(EmbeddedResourceProvider).Assembly;
+            var resourceStream = assembly.GetManifestResourceStream($"{assembly.GetName().Name}.Resources.{fileName}");
+
+            using var reader = new StreamReader(resourceStream, Encoding.UTF8);
+
+            return reader.ReadToEnd();
```

Comment:
This can be async.

Reply:
Any reason for that? Its an internal class method that is used in one place. Check where it is used (in `Lazy`). I can make it async, but in its usage there will be typical `.GetAwaiter().GetResult()` added. What for?